prospero-common/src/main/resources/prospero-known-combinations.yaml

```diff
@@ -1,7 +1,6 @@
 ---
 - name: "wildfly"
-  location: "org.wildfly:wildfly-galleon-pack"
-  packages: []
+  galleonConfiguration: "classpath:wildfly-provisioning.xml"
```

Comment:
Should the `wildfly-provisioning.xml` be part of the PR?

Reply:
It should be :).